<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VShaleDialog</class>
 <widget class="QDialog" name="VShaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>804</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Gamma Ray Max Log:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbGRMin"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Blocking Mode:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="leTertiaryRocks">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>VSHT</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="cbTertiaryRocks">
       <property name="text">
        <string>VSH Tertiary Rocks (Larionov):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Input Gamma Ray Log:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbGR"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Gamma Ray Index:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="leIGR">
       <property name="text">
        <string>IGR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Wells:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="cbSteiber">
       <property name="text">
        <string>VSH Steiber:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="leOlderRocks">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>VSHO</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="cbOlderRocks">
       <property name="text">
        <string>VSH Older Rocks (Larionov):</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QCheckBox" name="cbClavier">
       <property name="text">
        <string>VSH Clavier:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="leSteiber">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>VSHS</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="leClavier">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>VSHC</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbBlockingMode"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Gamma Ray Min Log:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Blocking Interfaces (Tops):</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QListWidget" name="lwTops">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="autoScrollMargin">
        <number>2</number>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QListWidget" name="lwWells">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cbGRMax"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VShaleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VShaleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTertiaryRocks</sender>
   <signal>toggled(bool)</signal>
   <receiver>leTertiaryRocks</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOlderRocks</sender>
   <signal>toggled(bool)</signal>
   <receiver>leOlderRocks</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSteiber</sender>
   <signal>toggled(bool)</signal>
   <receiver>leSteiber</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>474</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbClavier</sender>
   <signal>toggled(bool)</signal>
   <receiver>leClavier</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
